src/github.com/getlantern/flashlight/analytics/analytics.go

```diff
@@ -27,12 +28,15 @@ var (
 func Configure(cfg *config.Config, version string) {
 
 	if cfg.AutoReport != nil && *cfg.AutoReport {
-		analytics.Configure(TrackingId, version, cfg.Addr)
+		pubsub.Sub(pubsub.IP, func(ip string) {
+			log.Debugf("Got IP %v -- starting analytics", ip)
+			analytics.Configure(ip, TrackingId, version, cfg.Addr)
```

Comment:
Right -- I'm not familiar with how it works on the lantern-ui side. Does
that only make the call to analytics once the backend sets it up? If so
then we're golden.